consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/blockcreation/CliqueMiningCoordinator.java

```diff
@@ -33,6 +41,30 @@ public CliqueMiningCoordinator(
     this.miningTracker = miningTracker;
   }
 
+  @Override
+  protected void inSyncChanged(final boolean inSync) {
+    synchronized (this) {
+      final VoteTally validatorProvider =
+          miningTracker
+              .getProtocolContext()
+              .getConsensusState(CliqueContext.class)
+              .getVoteTallyCache()
+              .getVoteTallyAfterBlock(blockchain.getChainHeadHeader());
+      final boolean isValidator =
+          getCoinbase()
+              .filter(coinbase -> validatorProvider.getValidators().contains(coinbase))
+              .isPresent();
+
+      if (inSync && startMiningIfPossible() && isValidator) {
+        LOG.info("Resuming mining operations");
```

Comment:
Nit: could we not use "mining"?  I always associate that with PoW mining and nonce solving.  Is there a more different term like "sealing" "block production" or "validating" we could use?

Reply:
Done. I used block production for the log. For now, I keep the class called CliqueMiningCoordinator because it inherits from AbstractMiningCoordinator and implements MiningCoordinator. If we want to change that. this should be done in another PR. Because the impacts can be significant IMO